cmd.js

```diff
@@ -65,7 +65,8 @@ async function cmd (argv, banner = defaultBanner) {
       treeDebug: 'tree-debug',
       writeTicks: 'write-ticks',
       onPort: 'on-port',
-      P: 'onPort'
+      P: 'onPort',
+      collectDelay: 'on-timeout'
```

Comment:
```suggestion
      collectDelay: 'collect-delay'
```

Reply:
@goto-bus-stop - No problem. I'll update the arg in node-clinic to match.